alacritty/res/text.f.glsl

```diff
@@ -9,31 +9,29 @@ layout(location = 0, index = 1) out vec4 alphaMask;
 
 uniform sampler2D mask;
 
-void main()
-{
+void main() {
     if (backgroundPass != 0) {
-        if (bg.a == 0.0)
+        if (bg.a == 0.0) {
             discard;
+        }
 
         alphaMask = vec4(1.0);
         color = vec4(bg.rgb, 1.0);
-    } else {
-        if (fg.a != 0.0) {
-            // Color glyphs, like emojis.
-            vec4 glyphColor = texture(mask, TexCoords);
-            alphaMask = vec4(glyphColor.a);
-
-            // Revert alpha premultiplication.
-            if (glyphColor.a != 0) {
-                glyphColor.rgb = vec3(glyphColor.rgb / glyphColor.a);
-            }
+    } else if (int(fg.a) > 1) {
```

Comment:
This breaks as soon as another bitflag is added? Can we do the same we did for wide chars and use a constant and a bitwise and?